src/test/java/org/ecommerce/services/jpa/impl/OrderJpaServiceImplTest.java

```diff
@@ -216,6 +231,70 @@ void findAll() {
         );
     }
 
+    @DisplayName("Assess the findAll metadata")
+    @Test
+    void findAllAssesPagination() {
+        final List<Order> mockedList = List.of(order, order, order);
+        when(page.getContent()).thenReturn(mockedList);
+
+        when(orderDTOMapper.apply(any(Order.class))).thenReturn(orderDTO);
+        when(orderJpaRepository.findAll(any(Specification.class), any(Pageable.class)))
+                .thenReturn(page);
+
+        GetAllOrdersResponse ordersPage = orderJpaService.findAll(specificationParameters, pageable);
+        System.out.println(ordersPage.getPageSize());
```

Comment:
Pls remove `System.out.println` and use your logger instead

Reply:
You're right, i actually don't need it anymore https://github.com/algutierrez-griddynamics/e-commerce/pull/58/commits/96d81b0851dd3a812fbe4f53e649bd9bc0339bb3